### ISSUE TITLE
Hashing passwords with Bcrypt

### ISSUE THREAD

**Issue by nightfrost** [nightfrost/mytubeNET#3]
Hashing has to be implemented before we implement auth.

I think we need to split it into services before we start doing hashing as well...

Ressources:
https://jasonwatmore.com/post/2022/01/16/net-6-hash-and-verify-passwords-with-bcrypt
https://github.com/BcryptNet/bcrypt.net
